mllib/src/main/scala/org/apache/spark/ml/stat/KolmogorovSmirnovTest.scala

```diff
@@ -102,10 +102,11 @@ object KolmogorovSmirnovTest {
    */
   @Since("2.4.0")
   @varargs
-  def test(dataset: DataFrame, sampleCol: String, distName: String, params: Double*): DataFrame = {
+  def test(dataset: Dataset[_], sampleCol: String, distName: String, params: Double*)
```

Comment:
nit: This doesn't fit scala style; please get familiar with the style we use for multi-line function headers!  Just check out other parts of MLlib for examples.